includes/free-trial/class-wc-calypso-bridge-free-trial-orders-changes.php

```diff
@@ -0,0 +1,60 @@
+<?php
+/**
+ * Contains the logic for override WooCommerce > orders screen
+ *
+ */
+
+
+class WC_Calypso_Bridge_Free_Trial_Orders_Changes {
+	/**
+	 * The single instance of the class.
+	 *
+	 * @var object
+	 */
+	protected static $instance = null;
+
+	/**
+	 * Get class instance.
+	 *
+	 * @return object Instance.
+	 */
+	final public static function get_instance() {
+		if ( is_null( self::$instance ) ) {
+			self::$instance = new self();
+		}
+
+		return self::$instance;
+	}
+
+	public function __construct() {
+		if ( ! wc_calypso_bridge_is_ecommerce_trial_plan() ) {
+			return;
+		}
+
+		add_action('admin_print_footer_scripts', array($this, 'override_empty_state_cta_button_class') );
+	}
+
+	/**
+	 * Change the class of the button on the empty state of the orders screen.
+	 *
+	 */
+	public function override_empty_state_cta_button_class() {
+		if ( ! function_exists( 'get_current_screen' ) ) {
+			return;
+		}
+
+		$screen = get_current_screen();
+	
+		if ( $screen->id === 'edit-shop_order' ) {
+			?>
+			<script>
+				jQuery('.woocommerce-BlankState-cta.button')
```

Comment:
I think there were discussions on Woo getting rid of jQuery on preload. To be safe, I think it's better to use pure JS here. In fact, I can't help but ask ChatGPT to generate:

```
let woocommerceBlankStateButton = document.querySelector('.woocommerce-BlankState-cta.button');
woocommerceBlankStateButton.classList.add('button-secondary');
woocommerceBlankStateButton.classList.remove('button-primary');
```

Feel free to use/rename if it works 😸

Reply:
Work perfectly! 😃  

Changed in 8a9cc7e52ee946f807933e8ec260c52db1bcb103